portfolio/src/main/webapp/index.html

```diff
@@ -58,7 +58,22 @@ <h1 id="project">Servlet Tests</h1>
         <br>
         <input type="submit"/>
       </form>
+      
+      <p>Max Comments Shown:</p>
+      <select id="numComments">
+        <option name="1"> 1 </option>
+        <option name="2"> 2 </option>
+        <option name="3"> 3 </option>
+        <option name="4"> 4 </option>
+      </select>
+      <button onClick="fetchComments()">Load</button>
+
 
+      <form action="\delete-data" method="POST">
+        <p>Delete all comments</p>
+        <input type="submit"/>
```

Comment:
Potential next step might be to have a sign in section that only allows admins to delete comments. If that's interesting at all.